wizard/class-instant-articles-option.php

```diff
@@ -454,4 +454,37 @@ public function universal_sanitize_and_encode_handler( $payload ) {
 	public function sanitize_option_fields( $field_values ) {
 		return $field_values;
 	}
+
+	/**
+	 * Updates options from decoded map
+	 *
+	 * @param string $option_key to be returned.
+	 * @return array from a json decoded content.
+	 * @since 0.4
+	 */
+	public static function update_option( $option = array(), $option_key = null ) {
+		if ( ! isset( $option_key ) ) {
+			// Late Static Binding to use the const OPTION_KEY from the child class which called this function.
+			$option_key = static::OPTION_KEY;
+		}
+
+		wp_cache_delete ( 'alloptions', 'options' );
```

Comment:
On reflection, I wonder if this is even needed? We should trust WordPress to handle the "update_option" and delete the relevant caches itself, rather than forcing deletion of a bunch of stuff ourselves.